src/mqt/predictor/ml/helper.py

```diff
@@ -11,7 +11,16 @@
 from joblib import dump
 from qiskit import QuantumCircuit
 
-from mqt.predictor import utils
+from mqt.predictor import ml, utils
+
+
+def compile_prediction(qc: QuantumCircuit):
+    """Returns the compiled quantum circuit with the used combination of compilation options when the original
+    qasm circuit is provided as either a string or a file path."""
```

Comment:
that documentation string does not really match with the typing information in the function signature 🙃

Reply:
Adjusted the docu.